GVFS/GVFS.Installer.Mac/scripts/Distribution.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<installer-gui-script minSpecVersion="1">
+    <pkg-ref id="com.vfsforgit.pkg"/>
+    <pkg-ref id="com.git.pkg"/>
+    <options customize="never" allow-external-scripts="true"/>
+    <choices-outline>
+        <line choice="default">
+            <line choice="com.vfsforgit.pkg"/>
+            GIT_CHOICE_OUTLINE_PLACEHOLDER
+        </line>
+    </choices-outline>
+    <choice id="default"/>
+    <choice id="com.vfsforgit.pkg" visible="false">
+        <pkg-ref id="com.vfsforgit.pkg"/>
+    </choice>
+    <pkg-ref id="com.vfsforgit.pkg" version="VFSFORGIT_VERSION_PLACHOLDER" onConclusion="none">VFSFORGIT_PKG_NAME_PLACEHOLDER</pkg-ref>
+    GIT_CHOICE_ID_PLACEHOLDER
+    GIT_PKG_REF_PLACEHOLDER
+    <installation-check script="InstallationCheck()"/>
+    <script>
+        function InstallationCheck(prefix) 
+        {
+            var blockingProcesses = GetBlockingProcesses();
+            if (blockingProcesses.length > 0)
+            {
+                my.result.message = "Quit these processes and try again - " + blockingProcesses.join(", ") + ".";
+                my.result.type = 'Fatal';
+                return false;
+            }
+            
+            return true;
+        }
+        
+        function GetBlockingProcesses()
+        {
+            var watchList = [ "gvfs", "gvfs.mount", "git", "gitk" ];
```

Comment:
Have you tested this with `gitk`?  I believe the application name is actually "wish" and not "gitk".

Reply:
You are right `gitk` does launch `Wish`, will add it. (Our check still succeeds because it finds `gitk` as well, when running `ps` command.)
```
ps -ax | grep gitk
29744 ttys004    0:00.01 grep gitk
29673 ttys005    0:00.01 /bin/sh /usr/bin/wish /usr/local/git/bin/gitk --
29675 ttys005    0:02.82 /usr/bin/../../System/Library/Frameworks/Tk.framework/Versions/8.5/Resources/Wish.app/Contents/MacOS/Wish /usr/local/git/bin/gitk --```

Reply:
Added `wish`